packages/terra-core-docs/src/terra-dev-site/doc/tag/TagList.2.doc.mdx

```diff
@@ -0,0 +1,49 @@
+import { Badge } from 'terra-tag/package.json?dev-site-package';
+
+import TagListCollapsed from './example/TagListCollapsed?dev-site-example';
+
+import TagListSmall from './example/TagListSmall?dev-site-example';
+
+import TagPropsTable from 'terra-tag/lib/TagList?dev-site-props-table';
+
+<Badge />
+
+# Terra Tag List
+
+The Tag List is a subcomponent of main component `tag` which is used for grouping tabs together in a list to enable keyboard navigation using arrow keys. Navigation from one list to another can be done using `tab` key and within the list, navigation can be done using arrow keys, `right key` to move to next tag and `left key` to move to previous tag.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-tag-list`
+
+<!-- AUTO-GENERATED-CONTENT:START Peer Dependencies -->
+## Peer Dependencies
+
+This component requires the following peer dependencies be installed in your app for the component to properly function.
+
+| Peer Dependency | Version |
+|-|-|
+| react | ^16.8.5 |
+| react-dom | ^16.8.5 |
+
+<!-- AUTO-GENERATED-CONTENT:END -->
```

Comment:
we might need to add implementation details on both tag and taglist about page similar to this  https://github.com/cerner/terra-core/blob/ceabc03e855b802114d93f40b95594068b7c0503/packages/terra-core-docs/src/terra-dev-site/doc/badge/About.1.doc.mdx#L35

Reply:
https://github.com/cerner/terra-core/pull/3854/commits/8ded50f47258067e34fd8a01eed5eb3b47b92fd7#diff-a9cc2ccc1dba9de522f4dfa33689424aee764e7c6f91af3b23e597e3b897afd3